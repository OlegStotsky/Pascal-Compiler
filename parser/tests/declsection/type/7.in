Type
a = integer;
b = a;
c = b;