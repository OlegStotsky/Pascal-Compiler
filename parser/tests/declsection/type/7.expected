Syntax error at line 4, column 6 : BEGIN expected but EOF found
